website/content/en/preview/concepts/nodeclasses.md

```diff
@@ -244,6 +248,43 @@ kubelet:
   clusterDNS: ["10.0.1.100"]
 ```
 
+{{% alert title="Note" color="primary" %}}
+If you need to specify a field that isn't present in `spec.kubelet`, you can set it via custom [UserData]({{< ref "#specuserdata" >}}).
+For example, if you wanted to configure `maxPods` and `registryPullQPS` you would set the former through `spec.kubelet` and the latter through UserData.
+The following example achieves this with AL2023:
+
+```yaml
+apiVersion: karpenter.k8s.aws/v1
+kind: EC2NodeClass
+spec:
+  amiSelectorTerms:
+    - alias: al2023@latest
+  kubelet:
+    maxPods: 42
+  userData: |
+    apiVersion: node.eks.aws/v1alpha1
+    kind: NodeConfig
+    spec:
+      kubelet:
+        config:
+          # Configured through UserData since unavailable in `spec.kubelet`
+          registryPullQPS: 10
+```
+
+Note that when using the `Custom` AMIFamily you will need to specify fields **both** in `spec.kublet` and `spec.userData`.
+{{% /alert %}}
+
+{{% alert title="Warning" color="warning" %}}
+The Bottlerocket AMIFamily currently does not currently support the following fields:
```

Comment:
Can we make a note that these fields will be supported in the future, so do not set these fields if you do not want unexpected behavior when support for these fields are added with BR? I want to conserve the flexibility to add these post-v1 without it being breaking

Reply:
Aside: [timeless](https://developers.google.com/style/timeless-documentation) documentation is what we like to see, where feasible.

Reply:
Sounds good, I'll add a reference to the issue. I'll also remove the "currently" wording, though we will need to update this in the future regardless.